src/Stratis.Bitcoin.Features.SmartContracts/Wallet/SmartContractWalletController.cs

```diff
@@ -94,6 +120,16 @@ public IActionResult GetAccountAddresses(string walletName)
             }
         }
 
+        /// <summary>
+        /// Gets the balance at a specific wallet address in STRAT (or the sidechain coin).
+        /// This method gets the UTXOs at the address that the wallet can spend.
+        /// The function can be used to query the balance at a smart contract account address
+        /// supplied by /api/SmartContractWallet/SC-account-address.
```

Comment:
I'd remove all hardcoded references to API endpoints.

Reply:
This should do the trick :)

```suggestion
        /// supplied by <see cref="SmartContractWalletController.GetAccountAddresses"/>.
```